qdlt/qdltpluginmanager.cpp

```diff
@@ -134,108 +151,240 @@ QStringList QDltPluginManager::loadPluginsPath(QDir &dir)
 
 void QDltPluginManager::loadConfig(QString pluginName,QString filename)
 {
+    pMutex_pluginList->lock();
     for(int num=0;num<plugins.size();num++)
     {
         QDltPlugin *plugin = plugins[num];
         if(plugin->getName()==pluginName)
             plugin->setFilename(filename);
     }
+    pMutex_pluginList->unlock();
 }
 
 
 void QDltPluginManager::decodeMsg(QDltMsg &msg, int triggeredByUser)
 {
+    pMutex_pluginList->lock();
```

Comment:
this still slows down the overall performance of processing every single lock, or? Why has this been merged?
What's the performance impact on opening/viewing a bigger file?
@gobater  @Usha-Rajanna